docs/root/intro/arch_overview/observability/access_logging.rst

```diff
@@ -23,9 +23,10 @@ Periodic access logs
 --------------------
 
 If access log is enabled, then by default it will be reported to the configured sinks at the end of a TCP
-stream, or HTTP request. It is possible to extend this behavior and report  access logs periodically, and
-right after the TCP Proxy has successfully established a connection with the upstream. Reporting access logs
-right after upstream connection establishment does not depend on periodic reporting, and and the other way around.
+stream, or HTTP request. It is possible to extend this behavior and report access logs periodically, and
+right after the TCP Proxy has successfully established a connection with the upstream, or the HTTP router has
+successfully established TCP connection with the upstream .Reporting access logs right after upstream
```

Comment:
established TCP connection -> established a connection
(for HTTP/3 it's not a TCP connection)